### DISCUSSION TITLE
MyST rendering in VSCode

### DISCUSSION THREAD

**Discussion by stratisMarkou** [executablebooks/meta#1186] [Q&A]
MyST rendering in JupyterLab, for directives like theorems, lemmas and so on, is _awesome_! Is there a way to render these directives when viewing a notebook through VS Code? I've looked at existing VS Code extensions but couldn't find one that does this. Any pointers would be appreciated!

**Comment by ValMystletainn** [comment]
there is an [official extension](https://marketplace.visualstudio.com/items?itemName=ExecutableBookProject.myst-highlight)
however the extension is not updated for a long time
